<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListboxErrorMessageFormClass</class>
 <widget class="GUIFormBase" name="ListboxErrorMessageFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ListboxErrorMessageForm</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>101</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Page</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date/Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Code</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Level</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Message</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonClear">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="icon">
    <iconset resource="ListboxErrorMessage.qrc">
     <normaloff>:/Qtres/Clear.png</normaloff>:/Qtres/Clear.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCount">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>110</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>個数</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetPageStatus">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>411</width>
     <height>55</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Page0</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Page1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Page2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Page3</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonUpdate">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
   <property name="icon">
    <iconset resource="ListboxErrorMessage.qrc">
     <normaloff>:/Qtres/Update.png</normaloff>:/Qtres/Update.png</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ListboxErrorMessage.qrc"/>
 </resources>
 <connections/>
</ui>
